py-rse/git-cmdline.Rmd

```diff
@@ -10,6 +10,29 @@ These interfaces are actually wrappers around Git's original command-line interf
 which allows us to access all of Git's features.
 This lesson describes how the basic cycle of add, commit, push, and pull works at the command line.
 
+To illustrate with the Zipf's Law project, 
+we will start by putting our work up to this point in git.  
+Then, to illustrate the version control cycle, 
+we'll edit our, `plotcounts.py` script to produce a more useful plot.
+
+The `zipfs-law/` project directory, should currently look like this:
+```
+├── bin
+│   ├── book_summary.sh
+│   ├── countwords.py
+│   ├── collate.py
+│   ├── plotcounts.py
+│   └── mymodule.py
+├── data
+│   ├── anne_of_green_gables.txt
+│   ├── common_sense.txt
+|   └── ...
+└── results
```

Comment:
In the very latest version of #325, the three books that results are generated for are `results/emma.csv`, `results/dracula.csv` and `results/moby_dick.csv`.